Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -104,6 +104,7 @@ The gateway settings correspond to the RGW daemon settings.
 * `caBundleRef`: If specified, this is the name of the Kubernetes secret (type `opaque`) that
   contains additional custom ca-bundle to use. The secret must be in the same namespace as the Rook
   cluster. Rook will look in the secret provided at the `cabundle` key name.
+* `hostNetwork`: Controls whether the rgw pod may use the node network namespace. Doing so gives the pod access to the loopback device, services listening on localhost, and could be used to snoop on network activity of other pods on the same node.
```

Comment:
How about we just describe the behavior of the setting, and we can assume they know what host networking means?
```suggestion
* `hostNetwork`: Whether host networking is enabled for the rgw daemon. If not set, the network settings from the cluster CR will be applied.
```